agent/src/main/java/com/testsigma/agent/controllers/HomeController.java

```diff
@@ -57,7 +61,7 @@ public ResponseEntity<String> register(@PathVariable("uuid") String uuid,
       agentDTO.setHostName(hostName);
       agentDTO.setOsVersion(AgentService.getOsVersion());
       agentDTO.setAgentVersion(this.agentConfig.getAgentVersion());
-      agentDTO.setBrowserList(agentBrowserService.getBrowserList());
+      agentDTO.setBrowserList(getUniqueBrowsersList(agentBrowserService.getBrowserList()));
```

Comment:
@nixonakash01 Please change the browser list to a set instead

Reply:
it is in Set inside the function getUniqueBrowsersList @tarun-testsigma

Reply:
@nixonakash01 I meant change the `private List<AgentBrowser> browserList;` in AgentDTO to a Set

Reply:
If i change it to Set, then its broken and browsers list and os is not displayed in agents page. @tarun-testsigma

Reply:
we can keep it as List and the browsers list is obtained in the function where the variable is declared as Set and that is returned, so there will no duplicates. @tarun-testsigma

Reply:
Why do you want to convert it as Set and then convert it again to list ? If we want browserlist to be unique, it makes sense to make it a set

Reply:
if we make it a set then it is breaking the agents page, also even if its List, the value added to the List is Set and Set only obtains the available browsers list.